contrib/phrases/phrases.py

```diff
@@ -0,0 +1,158 @@
+# coding: utf8
+from __future__ import unicode_literals
+import sys
+
+import prodigy
+from prodigy.core import recipe_args
+from prodigy.components.db import connect
+from prodigy.components.sorters import Probability
+from prodigy.util import log, prints, split_string, set_hashes
+import requests
+import spacy
+import srsly
+
+
+API_URL = ""
+
+
+@prodigy.recipe('phrases.teach',
+    dataset=recipe_args["dataset"],
+    seeds=("One or more comma-separated seed terms", "option", "se", split_string),
+    threshold=("Similarity threshold for sense2vec", "option", "t", float),
+    batch_size=("Similarity threshold for sense2vec", "option", "bs", int),
```

Comment:
Sorry, missed this earlier!

```suggestion
    batch_size=("Batch size to use", "option", "bs", int),
```

But otherwise it's good to merge! Do you have a Twitter account we can credit when we share this on Twitter?